### PR TITLE
feat(website): add logo showcase - FRONT-3524

### PR DESCRIPTION
**Pull request by emeryro**
Add new pages on the website to showcase EC and EU logos (under "guidelines").

**Important:**
EC logo positive have been put in a dedicated folder (they were at the package root). This was to keep some consistency, and cleanup a little the structure.
Components and website pages using these logo have been updated, but it may have an impact if users were using the paths as is

### CODE REVIEW

File: src/website/src/website-components/Logo/LogoCard.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
+@use 'sass:map';
+@use '@ecl/theme-dev/theme';
+
+%card {
+  align-items: flex-start;
+  border: 2px solid transparent;
+  box-sizing: border-box;
+  display: flex;
+  flex-basis: 16rem;
```
Comment (papegaill) [inline src/website/src/website-components/Logo/LogoCard.scss]: `14.5rem` to display them in 3 columns?

File: src/website/src/pages/ec/guidelines/logo/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: Logo
```
Comment (papegaill) [inline src/website/src/pages/ec/guidelines/logo/index.md]: plural,  Logo**S** ?

File: src/website/src/pages/eu/guidelines/logo/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+title: Logo
```
Comment (papegaill) [inline src/website/src/pages/eu/guidelines/logo/index.md]: plural?

**Review by papegaill — COMMENTED** [review]
Nice! I have only 2 small suggestions.

File: src/website/src/website-components/Logo/LogoCard.scss (hunk 1 above)
Comment (emeryro) [reply src/website/src/website-components/Logo/LogoCard.scss]: @papegaill on my screen they are displayed on 4 columns, so do you mean making them bigger, in 3 columns?

Comment (papegaill) [reply src/website/src/website-components/Logo/LogoCard.scss]: On my macbook pro screen @100%, they are displayed 2 columns
Screenshot 2022-01-20 at 14 22 28

**Review by papegaill — DISMISSED** [review]

**Pull request merged**